### PR TITLE
impr(quotes): Added 8 English Quotes

### PR DESCRIPTION
**Pull request by agentaubie**
### Checks

- [x] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [ ] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [ ] Make sure to include your GitHub username inside parentheses at the end of the PR title

Closes #

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38666,6 +38666,54 @@
       "source": "Jorge Mario Bergoglio",
       "length": 100,
       "id": 7029
+    },
+    {
+      "text": "Soap trusted you. I thought I could too. So, why in bloody hell, does Makarov know you?",
+      "source": "Cpt. Price - Call of Duty: Modern Warfare 3",
+      "length": 87,
+      "id": 7030
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38674]: Remove

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38666,6 +38666,54 @@
...
+    },
+    {
+      "text": "There are some things that you don't want to do and you pledge to yourself that you won't do, you forbid yourself, and then suddenly they happen all by themselves. You don't even have time to think about them, and they don't make it to the cognitive centers of the brain: they just happen and that's it, and you're left just watching yourself with surprise, and convincing yourself that it wasn't your fault, it just happened all by itself.",
+      "source": "Metro 2033",
+      "length": 440,
+      "id": 7031
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38680]: Break it up a little. All this text is just 2 sentences.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/7805635281

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -38666,6 +38666,54 @@
...
+    },
+    {
+      "text": "He who is brave and patient enough to peer into the darkness his whole life will be first to see a flicker of light in it.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38683]: comma after `life`

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -38666,6 +38666,54 @@
...
+      "text": "I survived because the fire inside me burned brighter than the fire around me.",
+      "source": "Fallout: New Vegas - Joshua Graham",
+      "length": 78,
+      "id": 7034
+    },
+    {
+      "text": "Wanna break from the ads? If you tap now to watch a short video, you will receive 30 minutes of ad-free music… Yes, Really! If you tap now to watch a short video, you will receive 30 minutes of ad-free music!",
+      "source": "Spotify",
+      "length": 208,
+      "id": 7035
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38704]: Remove

File: frontend/static/quotes/english.json [hunk 5]
```diff
@@ -38653,19 +38653,55 @@
       "text": "If there is magic on this planet, it is contained in water.",
       "source": "Loren Eiseley",
       "length": 59,
-      "id": 7027
+      "id": 7026
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38656]: As you can see in the quote above this one, `7026` already exists. You need to up each id youve added by one

File: frontend/static/quotes/english.json [hunk 6]
```diff
@@ -38653,19 +38653,55 @@
...
+      "id": 7027
     },
     {
       "text": "Life is in a land full of thorns and weeds, there is always a space in which the good seed can grow.",
       "source": "Jorge Mario Bergoglio",
       "length": 100,
+      "id": 7028
+    },
+    {
+      "text": "There are some things… you pledge to yourself that you won't do, you forbid yourself... And then suddenly they happen all by themselves… and you're left just watching yourself with surprise, and convincing yourself that it wasn't your fault…",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38671]: Replace all ellipsis characters with triple dot (`...`) for consistency. Also remove first ellipsis and replace 3rd with a comma

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/7845382020

File: frontend/static/quotes/english.json [hunk 7]
```diff
@@ -38666,6 +38666,42 @@
       "source": "Jorge Mario Bergoglio",
       "length": 100,
       "id": 7029
+    },
+    {
+      "text": "There are some things you pledge to yourself that you won't do, you forbid yourself. Then suddenly they happen all by themselves, and you're left just watching yourself with surprise, and convincing yourself that it wasn't your fault.",
+      "source": "Metro 2033",
+      "length": 233,
```
Comment (NipulM) [inline frontend/static/quotes/english.json:38673]: The length of this text is 234, not 233.

File: frontend/static/quotes/english.json [hunk 8]
```diff
@@ -38666,6 +38666,42 @@
...
+    {
+      "text": "We know little about love. Love is like a pear. A pear is sweet and has a distinct shape. Try to define the shape of a pear.",
+      "source": "Dandelion - The Witcher: The Time of Contempt",
+      "length": 124,
+      "id": 7034
+    },
+    {
+      "text": "All roads lead back to one's home. Not your birthplace maybe, but home.",
+      "source": "Fallout New Vegas - Ulysses",
+      "length": 70,
```
Comment (NipulM) [inline frontend/static/quotes/english.json:38703]: The length of this text is 71, not 70.

**Pull request closed without merging**